anyway/infographics_utils.py

```diff
@@ -188,21 +193,19 @@ def get_accident_count_by_severity(self, location_info, location_text, start_tim
 
 
 class MostSevereAccidentsTableWidget(Widget):
-
     def __init__(self, request_params: RequestParams):
-        Widget.__init__(self, request_params)
-        self.widget_id = WidgetId.most_severe_accidents_table
-        self.name = self.widget_id.name
+        Widget.__init__(self, request_params, WidgetId.most_severe_accidents_table)
         self.rank = 2
 
     def generate_items(self) -> None:
-        self.items = self.get_most_severe_accidents_table(
+        self.items = MostSevereAccidentsTableWidget.get_most_severe_accidents_table(
```

Comment:
The name seems redundant. I suggest `MostSevereAccidentsTableWidget.prepare_table()` or something like that.